webpack.config.js

```diff
@@ -40,5 +40,6 @@ module.exports = {
 		upe_onboarding_wizard: './client/upe-onboarding-wizard/index.js',
 		upe_opt_in_banner: './client/entrypoints/upe-opt-in-banner/index.js',
 		upe_settings: './client/settings/index.js',
+		payment_gateways: './client/payment-gateways/index.js',
```

Comment:
We started placing entrypoints in the `entrypoints` directory - see pc4etw-jM-p2

Reply:
👍 Moved my files to `./client/entrypoints/payment-gateways`